src/query/generated/proto/rpcpb/query.proto

```diff
@@ -65,10 +65,20 @@ message FetchOptions {
 }
 
 message RestrictFetchOptions {
+	RestrictFetchType RestrictFetchType = 1;
+	RestrictFetchTags RestrictFetchTags = 2;
```

Comment:
I know this may be contentious since I'm dropping backcompat here, but I figure query hosts are pretty trivial to upgrade; can make it more back compatible though

Reply:
Hm, this is probably ok. I don't think there's many users using either of these fields.

Maybe we should make it fields `3, 4` instead of `1, 2` though so it doesn't try to unpack something during a deploy and it unpacks a weird value it's not expecting?